ble_controller/include/ble_controller.h

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright (c) 2018 - 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+
+/**
+ * @file ble_controller.h
+ *
+ * @defgroup ble_controller BLE Controller API
+ *
+ * The main APIs needed to configure, enable, and use the BLE Controller.
+ * @{
+ */
+
+
+#ifndef BLE_CONTROLLER_H__
+#define BLE_CONTROLLER_H__
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+#include <stdint.h>
+#include "nrf.h"
+#include "nrf_errno.h"
+
+
```

Comment:
I find it a bit weird that we changed the defaults that we have always used in SoftDevices.
I'm not saying we should change them now. but I think we should get them back before we do a proper release.
Is it OK to create a jira issue to revert these? to the s132 defaults.
Master link count should be 3, event length should be 3 * 1.25 ms.

Reply:
We can take an internal discussion whether or not to revert anything. The default master link count=3 is valid -- just to show off that we can do more than 1. However, 3*1.25ms event length is not long enough when using coded phy.

I don't expect us to maintain full compatibility with what has been the defaults for the SoftDevice, much of this will be controlled through Kconfig anyway (when used in NCS).